dags/config/green/indicateurs_sip_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "schema": {"name" : "indicateurs_sip", "postgres_schema_creation_sql_path" : "sql/indicateurs_sip/schema.sql"},
+  "tables": [
+    {"name": "catheter"   , "postgres_table_creation_sql_path" : "sql/indicateurs_sip/tables/catheter_schema.sql"   , "copy_conf": {"src_s3_bucket" :  "green-prd", "src_s3_key" :  "released/indicateurssip/catheter/catheter.csv"      , "dts_postgres_tablename" : "catheter"}}   ,
```

Comment:
est ce qu'on a vraiment besoin de `"copy_conf"` et `"name"` ? 
peut-être une version simplifié:
```
tables": [
    {
      "table_name": "catheter",
      "creation_sql_path": "sql/indicateurs_sip/tables/catheter_schema.sql",
      "src_s3_bucket": "green-prd",
      "src_s3_key": "released/indicateurssip/catheter/catheter.csv"
    },
...
]
Même la nomeclature des clés c'est qq chose qu'on peut améliorer dans le futur mais pas urgent c'est juste lorsque j'ai vu `s3_key` et que je trouve un path pour un csv ca ma mélanger un peu
```

Reply:
s3_key j'ai utiliser la meme nomenclature que le s3Hook:

https://airflow.apache.org/docs/apache-airflow/1.10.6/_api/airflow/hooks/S3_hook/index.html

Sinon je suis d'accord que j'aimerai améliorer la config. Je pensait peut-être just enlever les creation de sql et garder ca séparer. Mais l'aventage de l'avoir ensemble c'est d'être sur que pour chanque table dans la conf, on a écrit un fichier pour la création de table. 

Laissez-moi savoir ce que vous pensez

Reply:
En fait les paths dans s3 (minio) ce sont des s3 keys donc c'est clair je trouve. s3_key c'est standard pour représenter le path d'un objet dans S3, c'est le nom de variable que je vois tout le temps en fait.

Reply:
As-tu testé en local si le dag se crée? Car dags.py va essayer de parser ce fichier et vu qu'il a pas la même structure que les autres je pense que ça va planter.

Reply:
nice c'st bon à savoir --> `key ([str])– S3 key that will point to the file` ils ont trouvé que le mot key pour representer un path ca m'a mélanger excuser

Reply:
C'est parce que S3 c'est un key-value object store. En fait y a pas vraiment de paths dans s3. Tout objet est identifié par une clé, c'est juste que la clé peut contenir le caractère \ et le UI te l'affiche comme un path.

Reply:
C'est vrai j'avait pas penser a ca. Je pense que tu as raison ça va planter. je vais essayer.

Reply:
J'ai tester en local et il ne pouvait pas trouver le fichier. Je les ai séparé et ammener dans le fichier .py ou les DAGs sont déclaré.